pkg/ctl/cmdutils/filter/iamserviceaccount_filter_test.go

```diff
@@ -54,7 +60,72 @@ var _ = Describe("iamserviceaccount filter", func() {
 				"sa/test3",
 			)).To(BeTrue())
 		})
-		// TODO test SetExcludeExistingFilter() which requires mocking different kubernetes functions
+
+		It("exclude existing stacks works correctly", func() {
```

Comment:
Can we fix the `It` strings?

Reply:
sorry, I don't understand what you mean by "fix" here

Reply:
Ah nevermind, I think I misread they were left copy pasted from a different `It`, sorry!